### PR TITLE
Created Session history cards for History Page

### PR DESCRIPTION
**Pull request by bryanansong**
## Description
I created cards at the bottom of the history page that show the past sessions that a user has logged. In future Pull Requests, I will make each session render based on fetches I make to the backend.

<what this pull request is doing>
<what will be done in later PRs and not included here>

## Test Plan
Screenshot 2024-06-30 at 1 06 44 AM

<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>

### CODE REVIEW

File: frontend/src/components/SessionHistory/SessionHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    return `${day}, ${dateNum}${suffix} ${month}`;
+  };
+  return (
+    <div className={styles.container}>
+      <p className={styles.cardTitle}>
+        {session?.workoutTemplate?.name || "Full Body Day"}
+        <span className={styles.dateAndDuration}>
+          <span>{formatDate(session.date)}</span>
+          <span>{`${session.duration / 60} mins`}</span>
```
Comment (cherieliu) [inline frontend/src/components/SessionHistory/SessionHistory.tsx:74]: noob question - does dividing by an integer (60 in this case) round the quotient value to an integer as well?

File: frontend/src/components/SessionHistory/SessionHistory.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        : dateNum === 3 || dateNum === 13 || dateNum === 23
+        ? "rd"
+        : "th";
+
+    return `${day}, ${dateNum}${suffix} ${month}`;
+  };
+  return (
+    <div className={styles.container}>
+      <p className={styles.cardTitle}>
+        {session?.workoutTemplate?.name || "Full Body Day"}
```
Comment (Wang96) [inline frontend/src/components/SessionHistory/SessionHistory.tsx:71]: Is session or workoutTemplate or name nullable here? I am thinking we might need to return null immediately if session is null, otherwise you might just see a card with title "Full Body Day" but nothing in the content.

File: frontend/src/components/SessionHistory/SessionHistory.tsx (hunk 1 above)
Comment (bryanansong) [reply frontend/src/components/SessionHistory/SessionHistory.tsx:74]: Division in JavaScript returns a float. Thanks for bringing it to my attention. I should probably round this figure to prevent having several decimal places

File: frontend/src/components/SessionHistory/SessionHistory.tsx (hunk 2 above)
Comment (bryanansong) [reply frontend/src/components/SessionHistory/SessionHistory.tsx:71]: Session, WorkoutTemplate and Name would not be null in this situation. I did it this way to have some placeholder values to work with. It would be good to replace that line with `{ session.workoutTemplate.name }` instead
